configuration/items.md

```diff
@@ -310,17 +313,67 @@ The following guidelines apply to user-added icon files:
+
+If the value contains only two segments, the first segment is assumed to be the icon source and the second the icon name.
+In this case, the icon set is to "classic" by default.
+This is used in particular when the icon source only provides a single set of icons.
+As an example, "&lt;material:favorite&gt;" references the "favorite" icon from the Material icons.
+"&lt;oh:switch&gt;" references the "switch" icon from the openHAB classic icon set.
+
+If the value contains only one segment, it is assumed to be the name of an icon from the openHAB classic icon set.
+As an example, "&lt;switch&gt;" references the "switch" icon from the openHAB classic icon set.
+
+Here are few examples showing the different options:
+
+```java
+Switch Livingroom_Light "Livingroom Ceiling Light" <switch>
+Switch Parent_Bedroom_Light "Parent Bedroom Light" <oh:switch>
+Switch Child_Bedroom_Light "Child Bedroom Light" <oh:classic:switch>
+Switch Bathroom_Light "Bathroom Light" <material:lightbulb>
+Switch Kitchen_Light "Kitchen Light" <f7:lightbulb>
+Switch Garage_Light "Garage Light" <if:mdi:lightbulb>
+```
+
+Here is the list of available icon sources and how they are supported by the major user interfaces.
+
+| Source name          | Source description                    | Main UI    | Basic UI                          | Android app       | iOS app           |
+| -------------------- | ------------------------------------- | ---------- | --------------------------------- | ----------------- | ----------------- |
+| `oh`                | Icons provided via the openHAB server | Supported  | Supported                         | Supported         | Supported         |
+| `material`         | Material icons                        | Supported  | Supported                         | Not yet supported | Not yet supported |
+| `f7`                | Framework7 icons                      | Supported  | Not yet supported                 | Not yet supported | Not yet supported |
+| `if` or `iconify` | iconify icons                         | Supported  | Supported but needs to be enabled | Not yet supported | Not yet supported |
+
+You can find a list of [all Material icons here](https://fonts.google.com/icons?icon.set=Material+Icons).
+
+You can find a list of [all Framework7 icons here](https://framework7.io/icons/).
+
+You can find a list of [all iconify icons here](https://icon-sets.iconify.design/).
+Please note that the iconify option requires Internet connectivity on the client to access the external API.
+The WEB browser will cache the retrieved icons to limit the requests and speed up the rendering.
+Certain user interfaces provide a setting to enable this option, e.g. Basic UI disables this option by default but allows it to be enabled.
+
 #### Dynamic Icons
 
-Some icons are dynamically selected by openHAB depending on the Item's state.
+When and only when the icon source is openHAB, some icons are dynamically selected by openHAB depending on the Item's state.
```

Comment:
I think "if" is more applicable, isn't it?

Reply:
Changed